### PR TITLE
alt setup

### PR DESCRIPTION
**Pull request by drcandacemakedamoore**
The environment works for everything! But unfortunately my machine would not let me build it in the instructed way, so I'm adding an alternative that worked for me

### CODE REVIEW

File: learners/setup.md [hunk 1]
```diff
@@ -69,6 +69,12 @@ After the installation is complete, activate the environment:
 ```bash
 mamba activate medical_image_proc
 ```
+Note: In the case that this setup does not work due to conflicting previous conda or mamba configurations on your machine, you can try cloning the repository, navigating into it, and then building the environment there.
+
+The cloning step is as follows: 
+```bash
+git clone https://github.com/esciencecenter-digital-skills/medical-image-processing.git
```
Comment (gcroci2) [inline learners/setup.md:76]: What was the issue you encountered specifically? At which point did the setup break for you?
If it's here: `mamba env create -f https://github.com/esciencecenter-digital-skills/medical-image-processing/blob/main/learners/environment.yml` and it is due to mamba installation you can try to change `mamba` to `conda` (this is also indicated in the instructions, line 47). 

I don't understand how cloning the repository can solve any possible conda/mamba conflicts (after you clone it, you need to use conda or mamba conda anyway to create the env, and if the system has conflicts they will still be there).

**Review by gcroci2 — CHANGES_REQUESTED** [review]

File: learners/setup.md (hunk 1 above)
Comment (drcandacemakedamoore) [reply learners/setup.md:76]: I tried changing the mamba for conda, and it still did not work. Something in my setup just does not work when I pull things this way, although I have no idea why. What did work was cloning the repo and going inside of it. Maybe the issue is not mamba/conda related, but I worked on this for a while, and it was a reproducible problem on my machine, at least the day I worked on it.

**Comment by drcandacemakedamoore** [comment]
@gcroci2 I think I found the problem. the command you wrote for me just points to the html page, not the actual yaml file. If I use the file ie. ```mamba env create -f https://raw.githubusercontent.com/esciencecenter-digital-skills/medical-image-processing/main/learners/environment.yml```, everything works just fine. If it works this way for you we can just change it to this. The traceback was mostly uninformative, and I figured it out another way, but maybe it says something to you: # >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\conda\exception_handler.py", line 17, in __call__
        return func(*args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\conda_env\cli\main.py", line 56, in do_call
        exit_code = getattr(module, func_name)(arguments, parser)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\conda\notices\core.py", line 119, in wrapper
        return_value = func(*args, **kwargs)
                       ^^^^^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\conda_env\cli\main_create.py", line 107, in execute
        spec = specs.detect(
               ^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\conda_env\specs\__init__.py", line 72, in detect
        if spec.can_handle():
           ^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\conda_env\specs\yaml_file.py", line 19, in can_handle
        self._environment = env.from_file(self.filename)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\conda_env\env.py", line 191, in from_file
        return from_yaml(yamlstr, filename=filename)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\conda_env\env.py", line 158, in from_yaml
        data = yaml_safe_load(yamlstr)
               ^^^^^^^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\conda\common\serialize.py", line 45, in yaml_safe_load
        return _yaml_safe().load(string)
               ^^^^^^^^^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\ruamel\yaml\main.py", line 434, in load
        return constructor.get_single_data()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\ruamel\yaml\constructor.py", line 119, in get_single_data
        node = self.composer.get_single_node()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\ruamel\yaml\composer.py", line 76, in get_single_node
        document = self.compose_document()
                   ^^^^^^^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\ruamel\yaml\composer.py", line 102, in compose_document
        self.parser.get_event()
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\ruamel\yaml\parser.py", line 168, in get_event
        self.current_event = self.state()
                             ^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\ruamel\yaml\parser.py", line 253, in parse_document_end
        token = self.scanner.peek_token()
                ^^^^^^^^^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\ruamel\yaml\scanner.py", line 188, in peek_token
        self.fetch_more_tokens()
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\ruamel\yaml\scanner.py", line 288, in fetch_more_tokens
        return self.fetch_value()
               ^^^^^^^^^^^^^^^^^^
      File "C:\Users\CandaceMoore\miniconda3\Lib\site-packages\ruamel\yaml\scanner.py", line 657, in fetch_value
        raise ScannerError(
    ruamel.yaml.scanner.ScannerError: mapping values are not allowed here
      in "<unicode string>", line 231, column 75:
         ... age-responsive" style="word-wrap: break-word;">
                                             ^ (line: 231)

`$ C:\Users\CandaceMoore\miniconda3\Scripts\mamba create -f https://github.com/esciencecenter-digital-skills/medical-image-processing/blob/main/learners/environment.yml`

  environment variables:
             ACLOCAL_PATH=C:\Users\CandaceMoore\AppData\Local\Programs\Git\mingw64\share\aclocal
                          ;C:\Users\CandaceMoore\AppData\Local\Programs\Git\usr\share\aclocal
                 CIO_TEST=<not set>
  CONDA_AUTO_UPDATE_CONDA=false
        CONDA_DEFAULT_ENV=base
                CONDA_EXE=C:/Users/CandaceMoore/miniconda3/Scripts/conda.exe
             CONDA_PREFIX=C:\Users\CandaceMoore\miniconda3
    CONDA_PROMPT_MODIFIER=(base)
         CONDA_PYTHON_EXE=C:/Users/CandaceMoore/miniconda3/python.exe
               CONDA_ROOT=C:\Users\CandaceMoore\miniconda3
              CONDA_SHLVL=1
           CURL_CA_BUNDLE=<not set>
                  EXEPATH=C:\Users\CandaceMoore\AppData\Local\Programs\Git\bin
                 HOMEPATH=\Users\CandaceMoore
                 INFOPATH=C:\Users\CandaceMoore\AppData\Local\Programs\Git\mingw64\local\info;C:
                          \Users\CandaceMoore\AppData\Local\Programs\Git\mingw64\share\info;C:\U
                          sers\CandaceMoore\AppData\Local\Programs\Git\usr\local\info;C:\Users\C
                          andaceMoore\AppData\Local\Programs\Git\usr\share\info;C:\Users\Candace
                          Moore\AppData\Local\Programs\Git\usr\info;C:\Users\CandaceMoore\AppDat
                          a\Local\Programs\Git\share\info
               LD_PRELOAD=<not set>
                  MANPATH=C:\Users\CandaceMoore\AppData\Local\Programs\Git\mingw64\local\man;C:\
                          Users\CandaceMoore\AppData\Local\Programs\Git\mingw64\share\man;C:\Use
                          rs\CandaceMoore\AppData\Local\Programs\Git\usr\local\man;C:\Users\Cand
                          aceMoore\AppData\Local\Programs\Git\usr\share\man;C:\Users\CandaceMoor
                          e\AppData\Local\Programs\Git\usr\man;C:\Users\CandaceMoore\AppData\Loc
                          al\Programs\Git\share\man
            ORIGINAL_PATH=C:\Users\CandaceMoore\AppData\Local\Programs\Git\mingw64\bin;C:\Users\
                          CandaceMoore\AppData\Local\Programs\Git\usr\bin;C:\Users\CandaceMoore\
                          bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows
                          \System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;C:\Progra
                          m Files\dotnet;C:\Ruby33-
                          x64\bin;C:\Users\CandaceMoore\AppData\Local\Programs\Python\Launcher;C
                          :\Users\CandaceMoore\AppData\Local\Microsoft\WindowsApps;C:\Users\Cand
                          aceMoore\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\Candace
                          Moore\AppData\Local\Programs\Git\cmd;C:\users\candacemoore\.local\bin;
                          C:\users\candacemoore\appdata\roaming\python\python312\scripts;C:\user
                          s\candacemoore\appdata\local\packages\pythonsoftwarefoundation.python.
                          3.11_qbz5n2kfra8p0\localcache\local-packages\python311\scripts
                     PATH=C:\Users\CandaceMoore\miniconda3;C:\Users\CandaceMoore\miniconda3\Libr
                          ary\mingw-
                          w64\bin;C:\Users\CandaceMoore\miniconda3\Library\usr\bin;C:\Users\Cand
                          aceMoore\miniconda3\Library\bin;C:\Users\CandaceMoore\miniconda3\Scrip
                          ts;C:\Users\CandaceMoore\miniconda3\bin;C:\Users\CandaceMoore\minicond
                          a3\condabin;C:\Users\CandaceMoore\bin;C:\Users\CandaceMoore\AppData\Lo
                          cal\Programs\Git\mingw64\bin;C:\Users\CandaceMoore\AppData\Local\Progr
                          ams\Git\usr\local\bin;C:\Users\CandaceMoore\AppData\Local\Programs\Git
                          \usr\bin;C:\Users\CandaceMoore\AppData\Local\Programs\Git\usr\bin;C:\U
                          sers\CandaceMoore\AppData\Local\Programs\Git\mingw64\bin;C:\Users\Cand
                          aceMoore\AppData\Local\Programs\Git\usr\bin;C:\Users\CandaceMoore\bin;
                          C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\Sys
                          tem32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;C:\Program Fi
                          les\dotnet;C:\Ruby33-
                          x64\bin;C:\Users\CandaceMoore\AppData\Local\Programs\Python\Launcher;C
                          :\Users\CandaceMoore\AppData\Local\Microsoft\WindowsApps;C:\Users\Cand
                          aceMoore\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\Candace
                          Moore\AppData\Local\Programs\Git\cmd;C:\users\candacemoore\.local\bin;
                          C:\users\candacemoore\appdata\roaming\python\python312\scripts;C:\user
                          s\candacemoore\appdata\local\packages\pythonsoftwarefoundation.python.
                          3.11_qbz5n2kfra8p0\localcache\local-packages\python311\scripts;C:\User
                          s\CandaceMoore\AppData\Local\Programs\Git\usr\bin\vendor_perl;C:\Users
                          \CandaceMoore\AppData\Local\Programs\Git\usr\bin\core_perl
          PKG_CONFIG_PATH=C:\Users\CandaceMoore\AppData\Local\Programs\Git\mingw64\lib\pkgconfig
                          ;C:\Users\CandaceMoore\AppData\Local\Programs\Git\mingw64\share\pkgcon
                          fig
PKG_CONFIG_SYSTEM_INCLUDE_PATH=C:/Users/CandaceMoore/AppData/Local/Programs/Git/mingw64/include
PKG_CONFIG_SYSTEM_LIBRARY_PATH=C:/Users/CandaceMoore/AppData/Local/Programs/Git/mingw64/lib
             PSMODULEPATH=C:\Program Files\WindowsPowerShell\Modules;C:\Windows\system32\Windows
                          PowerShell\v1.0\Modules
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : base
    active env location : C:\Users\CandaceMoore\miniconda3
            shell level : 1
       user config file : C:\Users\CandaceMoore\.condarc
 populated config files :
          conda version : 23.11.0
    conda-build version : not installed
         python version : 3.11.5.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=x86_64
                          __conda=23.11.0=0
                          __win=0=0
       base environment : C:\Users\CandaceMoore\miniconda3  (writable)
      conda av data dir : C:\Users\CandaceMoore\miniconda3\etc\conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\CandaceMoore\miniconda3\pkgs
                          C:\Users\CandaceMoore\.conda\pkgs
                          C:\Users\CandaceMoore\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\CandaceMoore\miniconda3\envs
                          C:\Users\CandaceMoore\.conda\envs
                          C:\Users\CandaceMoore\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/23.11.0 requests/2.31.0 CPython/3.11.5 Windows/10 Windows/10.0.22631 solver/libmamba conda-libmamba-solver/23.11.1 libmambapy/1.5.3
          administrator : False
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.
If you suspect this error is being caused by a malfunctioning plugin,
consider using the --no-plugins option to turn off plugins.

Example: conda --no-plugins install <package>

Alternatively, you can set the CONDA_NO_PLUGINS environment variable on
the command line to run the command without plugins enabled.

Example: CONDA_NO_PLUGINS=true conda install <package>

**Review by gcroci2 — APPROVED** [review]
Okay great, not sure why on my machine the not raw link was working. Anyway, all good now. I removed the additional comment on the cloning since it shouldn't be necessary now that you fixed the link.

**Pull request merged**